app/models/manageiq/providers/infra_manager/template.rb

```diff
@@ -2,7 +2,10 @@ class ManageIQ::Providers::InfraManager::Template < MiqTemplate
   default_value_for :cloud, false
 
   def self.eligible_for_provisioning
-    super.where(:type => %w(ManageIQ::Providers::Redhat::InfraManager::Template ManageIQ::Providers::Vmware::InfraManager::Template ManageIQ::Providers::Microsoft::InfraManager::Template))
+    super.where(:type => %w(ManageIQ::Providers::Redhat::InfraManager::Template
+                            ManageIQ::Providers::Vmware::InfraManager::Template
+                            ManageIQ::Providers::Microsoft::InfraManager::Template
+                            ManageIQ::Providers::Kubevirt::InfraManager::Template))
```

Comment:
@agrare I don't think it needs to be part of this PR but do we have a way to do this in a more pluggable way?

Reply:
😱 this is really nasty, looking at this and the `CloudManager::Template.eligible_for_provisioning` methods I think they include.....all of them?

Reply:
https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/application_controller/miq_request_methods.rb#L167-L179

Looks like all this really does is show Infra or Cloud templates, why don't they just do `ManageIQ::Providers::InfraManager::Template.all` or `ManageIQ::Providers::CloudManager::Template.all` ?

Reply:
From the Lifecycle -> Provisioning UI we show all templates, and that is not always an accurate list of what templates can be provisioned.  I think the best example here is that we did not support `Microsoft::InfraManager` template provisioning for a while after the templates were available in inventory.

Being able to select a provider's template because it is available in inventory without the supporting provisioning workflow/dialog would lead to UI errors.

From a pluggable solution we need to be able to ask each provider if they support template provisioning and then we can query the appropriate templates.